Fleece/MSVC_Compat.hh

```diff
@@ -38,6 +38,10 @@
     // WARNING: sizeof() will not work on this array since it's actually declared as a pointer.
     #define StackArray(NAME, TYPE, SIZE)    TYPE* NAME = (TYPE*)_malloca(sizeof(TYPE)*(SIZE))
 
+    #define strlcat strncat
```

Comment:
This is dangerous: strlcat is safer than strncat if the string overflows, so substituting strncat reopens the problem. I'd rather you found an implementation of strlcat to use.